From bf2066d4af15cb5723155e49504e413064fd8d83 Mon Sep 17 00:00:00 2001
From: spachatx <satish.chandrax.pachatu.prabhakara@intel.com>
Date: Mon, 28 Dec 2015 14:56:33 -0800
Subject: [PATCH 1/5] atomisp2: css2400 and 2401 irci_ecr-master_20151227_0200

Merge CSS2.1 for css2400 and 2401_legacy
for irci_ecr-master_20151208_2222 firmware tag

Change-Id: Ia3405b5f5c6cc64119a734be2432557148cdd1eb
Signed-off-by: spachatx <satish.chandrax.pachatu.prabhakara@intel.com>
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/92434
Reviewed-by: ManishX Gotefode <manishx.gotefode@intel.com>
Build: Dessert M BXT RVP (SC)
Build: Dessert M CHT FFD (SC)
Approver: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-15122
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/455603
---
 .../external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h   | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/ia_css_types.h              | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h       | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c           | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
index 375c72f..2d31e46 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
@@ -1,4 +1,4 @@
-/* Release Version: irci_ecr-master_20151212_1500 */
+/* Release Version: irci_ecr-master_20151227_0200 */
 /**
 Support for Intel Camera Imaging ISP subsystem.
 Copyright (c) 2010 - 2015, Intel Corporation.
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
index c2ed1c6..ac2d03c 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
@@ -1,4 +1,4 @@
-/* Release Version: irci_ecr-master_20151212_1500 */
+/* Release Version: irci_ecr-master_20151227_0200 */
 /**
 Support for Intel Camera Imaging ISP subsystem.
 Copyright (c) 2010 - 2015, Intel Corporation.
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
index da20e58..d033702 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
@@ -22,7 +22,7 @@ more details.
 #define __IA_CSS_VERSION_DATA_H
 
 
-#define CSS_VERSION_STRING "REL:20151212_50.6_2020; API:2.1.20.20; GIT:irci___#8f56dd693e4bd755b3c56e1383dc2c9ced11462d; SDK:; USER:viedifw; "
+#define CSS_VERSION_STRING "REL:20151227_52.0_0939; API:2.1.20.20; GIT:irci___#8f56dd693e4bd755b3c56e1383dc2c9ced11462d; SDK:; USER:viedifw; "
 
 
 #endif
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
index de8140e..279413c 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
@@ -47,10 +47,10 @@ struct fw_param {
 /* Warning: same order as SH_CSS_BINARY_ID_* */
 static struct firmware_header *firmware_header;
 
-/* The string STR(irci_ecr-master_20151212_1500) is a place holder
+/* The string STR(irci_ecr-master_20151227_0200) is a place holder
  * which will be replaced with the actual RELEASE_VERSION
  * during package generation. Please do not modify  */
-static const char *release_version = STR(irci_ecr-master_20151212_1500);
+static const char *release_version = STR(irci_ecr-master_20151227_0200);
 
 #define MAX_FW_REL_VER_NAME	300
 static char FW_rel_ver_name[MAX_FW_REL_VER_NAME] = "---";
-- 
1.9.1

